src/index.ts

```diff
@@ -12,7 +12,7 @@ const longPressStart = new CustomEvent('long-press-start')
 
 export const directiveOption = {
   bind (el: LongPressHTMLElement, binding: VNodeDirective, vnode: VNode) {
-    el.dataset.longPressTimeoutId = null
+    el.dataset.longPressTimeoutId = ''
```

Comment:
in that case, could u change it to `'0'`? and then i'll merge it :)

Reply:
Done (: